Lib/test/test_pdb.py

```diff
@@ -3232,6 +3232,37 @@ def test_pdb_issue_gh_127321():
     """
 
 
+def test_pdb_issue_gh_136057():
+    """See GH-136057
+    "step" and "next" commands should be able to get over list comprehensions
```

Comment:
Shouldn't it be that next steps over the list comp but step executes every iteration?